tests/tests/sorting.rs

```diff
@@ -185,15 +189,236 @@ fn sort_by_row_return_scores(mut conn: PgConnection) {
 
     let (plan, ) = "EXPLAIN (ANALYZE, FORMAT JSON) SELECT paradedb.score(id), * FROM paradedb.bm25_search WHERE description @@@ 'keyboard OR shoes' ORDER BY category LIMIT 5".fetch_one::<(Value,)>(&mut conn);
     eprintln!("{plan:#?}");
+
+    // Get the first plan node in the plans array
     let plan = plan
         .pointer("/0/Plan/Plans/0/Plans/0")
         .unwrap()
         .as_object()
         .unwrap();
+
     assert_eq!(plan.get("   Sort Field"), None);
     assert_eq!(plan.get("Scores"), Some(&Value::Bool(true)));
 }
 
+#[rstest]
+async fn test_incremental_sort_with_partial_order(mut conn: PgConnection) {
+    // Create the test table
+    sqlx::query(
+        r#"
+        CREATE TABLE sales (
+            id SERIAL,
+            sale_date DATE NOT NULL,
+            amount REAL NOT NULL,
+            description TEXT,
+            PRIMARY KEY (id, sale_date)
+        ) PARTITION BY RANGE (sale_date);
+        "#,
+    )
+    .execute(&mut conn)
+    .await
+    .unwrap();
+
+    // Create partitions
+    sqlx::query(
+        r#"
+        CREATE TABLE sales_2023_q1 PARTITION OF sales
+          FOR VALUES FROM ('2023-01-01') TO ('2023-04-01');
+        "#,
+    )
+    .execute(&mut conn)
+    .await
+    .unwrap();
+
+    sqlx::query(
+        r#"
+        CREATE TABLE sales_2023_q2 PARTITION OF sales
+          FOR VALUES FROM ('2023-04-01') TO ('2023-06-30');
+        "#,
+    )
+    .execute(&mut conn)
+    .await
+    .unwrap();
```

Comment:
This looks like it might be equivalent to the partitioned table fixture? https://github.com/paradedb/paradedb/blob/a1efcc576a9912e67a508c5e1c5635ba97599f46/tests/tests/fixtures/tables/partitioned.rs#L22-L60 if so, could you please use that?

Reply:
Done in PR #2466.